libbeat/_meta/config.reference.yml.tmpl

```diff
@@ -808,7 +806,7 @@ output.elasticsearch:
 
     # Configure escaping HTML symbols in strings.
     #escape_html: false
-
+{{end}}
 #================================= Paths ======================================
 
 # The home path for the beatname installation. This is the default base path
```

Comment:
Can you search these files for `beatname` and replace them all with a template variable.